### PR TITLE
WT-11297 Make incremental ID information durable always.

### PR DESCRIPTION
**Pull request by sueloverso**
The fix for this turned out to be easier than I originally thought because we already had the code in there for a forced stop of incremental backup.

### CODE REVIEW

**Comment by sueloverso** [comment]
Looks like there is fallout with this change as there are timestamp_abort failures missing data in the main database.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM!

**Review by keitharnoldsmith — COMMENTED** [review]
The code change LGTM. But I wonder if we need a forced checkpoint? It seems odd that we should need to explicitly write a new checkpoint to each unmodified dhandle in order to ensure that the backup ID information is durable.

I left a question about this on the ticket, since that seemed like the place to capture any discussion/explanation.

File: src/cursor/cur_backup.c [hunk 1]
```diff
@@ -205,20 +204,16 @@ __curbackup_close(WT_CURSOR *cursor)
...
+    }
+
+    /*
+     * For either a force stop or a full backup starting an incremental force a checkpoint so that
+     * the new information is visible in the metadata and old backup information does not reappear
+     * if we crash and restart.
+     */
+    if (F_ISSET(cb, WT_CURBACKUP_FORCE_STOP) ||
+      (F_ISSET(cb, WT_CURBACKUP_INCR) && cb->incr_src == NULL)) {
+        WT_TRET(session->iface.checkpoint(&session->iface, "force=1"));
```
Comment (pmacko86) [inline src/cursor/cur_backup.c:216]: Can you please add a comment explaining why this needs to be a forced checkpoint (as opposed to a normal checkpoint)?

**Review by pmacko86 — APPROVED** [review]
LGTM. Just like Keith, I have the same question about forced checkpoints.

**Comment by sueloverso** [comment]
evergreen retry

**Review by keitharnoldsmith — APPROVED** [review]
LGTM. Thanks for the last minute work on this one!

**Comment by pmacko86** [comment]
evergreen retry

**Pull request merged**